### DISCUSSION TITLE
Bookmark Categorization

### DISCUSSION THREAD

**Discussion by piyush-multiplexer** [dailydotdev/daily#540] [Feature Requests]
### Currently all bookmarks are stored in general space, it will be good to have feature like named spaces/folder to manage bookmark like Instagram post or browser.

**Comment by idoshamun** [comment]
Hi @piyush-multiplexer,

Thanks for the suggestion. Closing as it's a duplicate request, let's talk about it in the original thread :)
https://github.com/dailydotdev/daily/discussions/520
